src/main/java/com/puppycrawl/tools/checkstyle/checks/FinalParametersCheck.java

```diff
@@ -165,33 +166,40 @@ public int[] getRequiredTokens() {
 
     @Override
     public void visitToken(DetailAST ast) {
-        // don't flag interfaces
```

Comment:
this comment was added 23 years ago - https://github.com/checkstyle/checkstyle/commit/f313f31ab30308796ce76d167ad1f08e79e28fdd

not clear why, so it is ok to do interfaces validation now.